OracleFMWInfrastructure/dockerfiles/12.2.1.3/run_admin_server.sh

```diff
@@ -18,7 +18,8 @@ set_context() {
 set_context
 . ${scriptDir}/container-scripts/setEnv.sh ${scriptDir}/properties/domain.properties
 
-hostvolume=/Users/mriccell/temp
+# HOST volume where the domain home will be persisted
+hostvolume=/Users/mydirectory/temp
```

Comment:
I would suggest changing this to `/home/username/domain_home` as the current example is only used on macOS and I'd prefer to use a more Linux-specific example here. Also, the comment states "will be persisted" so the name "temp" seems incorrect to me.

Reply:
thanks for the suggestion, made changes